### ISSUE TITLE
[Homepage] Add a #content ID to the yt-hero-cta div, so that the accessibility skip link works

### ISSUE THREAD

**Issue by sboots** [ytgov/ckanext-yukondesign#143]
**Estimated hours: 1**

On the homepage, there isn't a div with a `content` ID, and so there's an accessibility error because the "Skip to main content" link doesn't work. (It's fine on all the other pages except the homepage.)

Can we add an `id="content"` attribute to the div with `class="yt-hero-cta"` near the top of the homepage? That will fix the homepage accessibility issue without affecting other pages.

<img width="1920" height="1032" alt="Image" src="https://github.com/user-attachments/assets/d62fa79c-300f-4bc9-ab80-30f3b5b2c9ed" />

**Comment by sboots** [comment]
Works great, thanks so much @willy1989cv ! 🙌
